imap/http_caldav.c

```diff
@@ -1672,6 +1672,8 @@ static int export_calendar(struct transaction_t *txn)
     modseq_t syncmodseq = 0;
     int unbind_flag = -1, unchanged_flag = -1;
     struct caldav_db *caldavdb = NULL;
+    bool no_declined = !!hash_lookup("noDeclined", &txn->req_qparams);
```

Comment:
nit: we support 'managed-id', so maybe this should be 'no-declined' for consistency or just 'declined' with 'declined=0' disabling including declined items, but I don't feel too strongly about it